test/services/atsData/PayeAtsTestData.scala

```diff
@@ -311,38 +310,45 @@ object PayeAtsTestData {
       DataHolder(
         Some(
           Map(
-            "scottish_starter_rate_income_tax_amount" -> Amount.gbp(380),
-            "scottish_starter_rate_income_tax" -> Amount.gbp(2000),
-            "scottish_basic_rate_income_tax_amount" -> Amount.gbp(2030),
-            "scottish_basic_rate_income_tax" -> Amount.gbp(10150),
-            "scottish_intermediate_rate_income_tax_amount" -> Amount.gbp(4080),
-            "scottish_intermediate_rate_income_tax" -> Amount.gbp(19430),
-            "scottish_higher_rate_income_tax_amount" -> Amount.gbp(12943),
-            "scottish_higher_rate_income_tax" -> Amount.gbp(31570),
-            "scottish_total_tax" -> Amount.gbp(19433)
+            "scottish_starter_rate_amount" -> Amount.gbp(380),
+            "scottish_starter_rate" -> Amount.gbp(2000),
+            "scottish_basic_rate_amount" -> Amount.gbp(2030),
+            "scottish_basic_rate" -> Amount.gbp(10150),
+            "scottish_intermediate_rate_amount" -> Amount.gbp(4080),
+            "scottish_intermediate_rate" -> Amount.gbp(19430),
+            "scottish_higher_rate_amount" -> Amount.gbp(12943),
+            "scottish_higher_rate" -> Amount.gbp(31570),
+            "scottish_total_tax" -> Amount.gbp(19433),
+
+            "basic_rate_income_tax_amount" -> Amount.gbp(380),
+            "basic_rate_income_tax" -> Amount.gbp(2000),
+            "higher_rate_income_tax_amount" -> Amount.gbp(2030),
+            "higher_rate_income_tax" -> Amount.gbp(10150),
+            "ordinary_rate_amount" -> Amount.gbp(4080),
+            "ordinary_rate" -> Amount.gbp(19430),
+            "upper_rate_amount" -> Amount.gbp(12943),
+            "upper_rate" -> Amount.gbp(31570),
+            "total_UK_income_tax" -> Amount.gbp(19433)
```

Comment:
Would it help the tests if Scottish and rUK values were different?

Reply:
agreed
changed the rUK totalIncome tax values